src/components/sections/launch-sequence.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Countdown from 'react-countdown';
+import SecondaryCta from '../secondary-cta';
+
+const hbpStartDate = new Date('12/31/2021, 23:59:59').getTime();
+let today = new Date();
+today = Date.now();
+const diff = hbpStartDate - today;
+
+const time = ({days, hours, minutes}) => {
+  var minutePadding = "";
+  var hourPadding = ""
+  if (minutes < 10) {
+    minutePadding = 0;
+  }
+  if(hours < 10) {
+    hourPadding = 0;
+  }
+  return <span>{days} : {hourPadding}{hours} : {minutePadding}{minutes} </span>;
+}
+const LaunchSequence = () => (
+  <div className="container ">
+    <div id="launch-sequence" className="launch__sequence">
+      <div className="launch__content">
+        <a className="launch__header">2022 Applications Close In</a>
+        <h1 className="launch__countdown">
+          <Countdown date={Date.now() + diff} 
```

Comment:
It would be ideal if this time counter updated every second. There should be some code in the livesite for the side counter that updates every second that you should be able to reference

Reply:
Okay! The counter can update every second, but I was following the design from figma, and that updated every minute

Reply:
Eh, i think it would be cooler, like wrocket ship countdown, for it to update by the second

Reply:
Agreed! That does look cooler tbh